HostLibraryTests/CMakeLists.txt

```diff
@@ -142,30 +143,39 @@ set(test_sources ${test_sources}
 if(TENSILE_USE_LLVM)
     find_package(LLVM REQUIRED CONFIG)
 
-    set(test_sources ${test_sources}
-        ContractionLibraryLoading_test.cpp
-        ContractionFitness_test.cpp
-        MultipleSolutionsPerSize_test.cpp
+    target_sources(TensileTests PRIVATE
         llvm/ArithmeticUnitPredicate_test.cpp
         llvm/CUEfficiencyPredicate_test.cpp
         llvm/DeterministicModePredicate_test.cpp
         llvm/KernelLanguagePredicate_test.cpp
         llvm/LLVMYAMLContraction_test.cpp
-        llvm/LibraryPerformance_test.cpp
-        )
+        llvm/LibraryPerformance_test.cpp)
 endif()
 
 if(Client IN_LIST TENSILE_COMPONENTS)
-    set(test_sources ${test_sources}
+    target_sources(TensileTests PRIVATE
         client/DataInitialization_test.cpp)
+    target_link_libraries(TensileTests PRIVATE Boost::filesystem)
```

Comment:
Removal of ${Boost_LIBRARIES} may introduce a  weakness here. If someone were to add a Boost lib requirement in the future, you would have multiple places to update link dependencies between both TensileTests and TensileTestLib

Reply:
@ellosel you can see in the blame I actually went the reverse to allow me to build and run a subset of tests without adding this dependency at all.  As Chris said the variable might allow less maintenance and there are some other boost components used.   As for my needs I am fine if you revert my changes on the link library name

Reply:
> @ellosel you can see in the blame I actually went the reverse to allow me to build and run a subset of tests without adding this dependency at all. As Chris said the variable might allow less maintenance and there are some other boost components used. As for my needs I am fine if you revert my changes on the link library name

Thanks for the input, I will take a look at the blame.

Reply:
Reverting this change to keep PR more narrowly scoped.